dashboard/app/views/home/_privacy.md

```diff
@@ -187,35 +189,43 @@ Whenever we share Student personal data, we hold our partners to privacy and sec
+
+We may also share the list of schools (solely the school identities, without any personal or identifying information about teachers or students) that use Code.org or have participated in our professional learning program with select partners, such as a local government department of education, so long as such partners agree to treat the information as confidential. 
 
-We also provide teachers other opportunities (such as when adding their school to the map of schools teaching Computer Science) to share their contact information (name, school, email) — at their option and under their control — with a Code.org Regional Partner so that teachers can be contacted about local professional learning workshops, resources, and events. 
+We also provide teachers other opportunities (such as when adding their school to the map of schools teaching Computer Science) to share their contact information (name, school, email) — at their option and under their control — with a Code.org Regional Partner so that teachers can be contacted about local professional learning workshops, resources, and events.
 
 ### We may share de-identified information on school or district usage with our Regional Partners
 
 Our Regional Partners will be able to see de-identified demographic information and usage of courses and professional learning programs for schools and districts in their area. They will not see names or contact information of any Teacher unless that Teacher chooses to share it as part of signing up for a local professional learning workshop or joining their professional learning program.
 
-### You (entirely under YOUR control) may share data with social networks
+### You (entirely under YOUR control) may choose to post some information publicly
+
+In certain cases, you may choose to post certain information about that is publicly accessible on our Website. This includes making public posts as a teacher on our forum, offering to be a volunteer if you are over the age of 18, or choosing to put information about your school or organization on a public map. When you are filling out a form to post information on our Website, we will always make it clear exactly what will get shared publicly. 
```

Comment:
Guessing it should be `post certain information about yourself that is` or `post certain information that is`